test/e2e/clustertask/start_test.go

```diff
@@ -1,4 +1,6 @@
+//go:build e2e
```

Comment:
[tektoncd/plumbing/](https://github.com/tektoncd/plumbing/blob/759f977c9f1b2afd2c88cafad4f23e944d8e60b2/tekton/images/test-runner/Dockerfile#L172) is still using GOLANGCI_VERSION 1.42.0
and `//go:build e2e` is supported by GOLANGCI_VERSION 1.44.0 and above 
ref: https://pkg.go.dev/cmd/go#hdr-Build_constraints

Reply:
As reference to this in documentation:
```
Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax. 
```

Reply:
Oh, I was wondering if ci would fail. Never mind  :+1: